components/captcha-challenge.tsx

```diff
@@ -128,17 +134,38 @@ const CaptchaChallenge: React.FC<{ phoneNumber: string }> = ({ phoneNumber }) =>
   const isLoading = captchaState.status === "loading" || createLoading || requestLoading
   const hasError = !isLoading && captchaState.status === "error"
 
+  const resetPage = () => {
+    clearCookies().finally(() => {
```

Comment:
Let's use async/await to he consistent across our codebases

Reply:
yep got fetch mixed up with axios. too many client http libs to keep track of lol